src/slog.c

```diff
@@ -91,8 +101,6 @@ slog(struct server *s, log_level level,
 	char line[256]; /* bounds are checked. */
 	int line_sz, ret;
 
-	if(level > s->cfg->verbosity) return; /* too verbose */
-
```

Comment:
I'm not sure that moving this makes a difference, the same amount of space is still pre-allocated on the stack.
Adding a check _before_ calling would save a little bit, at least the function prelude (IIRC).